extensions/wikia/CreateNewWiki/js/WikiBuilder.js

```diff
@@ -572,15 +627,16 @@ define('ext.createNewWiki.builder', ['ext.createNewWiki.helper', 'wikia.tracker'
 					data: {
 						wName: wikiName.val(),
 						wDomain: wikiDomain.val(),
-						wLanguage: wikiLanguage.find('option:selected').val(),
-						wVertical: verticalOption.val(),
+						wLanguage: wikiLanguage.val(),
+						wVertical: wikiVertical.val(),
 						wCategories: categories,
 						wAllAges: wikiAllAges.is(':checked') ? wikiAllAges.val() : null
 					},
 					token: preferencesToken
 				},
 				callback: function (res) {
 					throbberWrapper.stopThrobbing();
+					throbberWrapper.removeClass('creating-wiki');
 					cityId = res.cityId;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "cityId" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)